src/components/table/README.md

```diff
@@ -1 +1,190 @@
+    <tbody>
+        <tr>
+            <th scope="row">One.IU</th>
+            <td>One.IU was created to bring a modern app store experience to finding what you need at IU. With One.IU, you search for what you want to do, and click to launch it.</td>
+            <td><a href="#">one.iu.edu</a></td>
+        </tr>
+        <tr>
+            <th scope="row">Box</th>
+            <td>Box is a no-cost cloud storage and collaboration environment available to students, faculty, and staff.</td>
+            <td><a href="#">box.iu.edu</a></td>
+        </tr>
+    </tbody>
+</table>
+```
+
+#### Compact table
+
+The `rvt-table-compact` class will decrease the amount of vertical padding applied to each table cell. This variant can be useful in situations where you need to conserve vertical space when displaying large amounts of data.
+
+```
+<table class="rvt-table-compact">
+    <caption class="sr-only">Table example one</caption>
+    <thead>
+        <tr>
+            <th scope="col">Service</th>
+            <th scope="col">Description</th>
+            <th scope="col">URL</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th scope="row">One.IU</th>
+            <td>One.IU was created to bring a modern app store experience to finding what you need at IU. With One.IU, you search for what you want to do, and click to launch it.</td>
+            <td><a href="#">one.iu.edu</a></td>
+        </tr>
+        <tr>
+            <th scope="row">Box</th>
+            <td>Box is a no-cost cloud storage and collaboration environment available to students, faculty, and staff.</td>
+            <td><a href="#">box.iu.edu</a></td>
+        </tr>
+    </tbody>
+</table>
+```
+
+#### Cells table
+
+The `rvt-table-cells` class adds borders to all table cells which can help with vertical and horizontal visual scanning.
+
+```
+<table class="rvt-table-cells">
+    <caption class="sr-only">Table example one</caption>
+    <thead>
+        <tr>
+            <th scope="col">Service</th>
+            <th scope="col">Description</th>
+            <th scope="col">URL</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th scope="row">One.IU</th>
+            <td>One.IU was created to bring a modern app store experience to finding what you need at IU. With One.IU, you search for what you want to do, and click to launch it.</td>
+            <td><a href="#">one.iu.edu</a></td>
+        </tr>
+        <tr>
+            <th scope="row">Box</th>
+            <td>Box is a no-cost cloud storage and collaboration environment available to students, faculty, and staff.</td>
+            <td><a href="#">box.iu.edu</a></td>
+        </tr>
+    </tbody>
+</table>
+```
+
+## Accessibility (a11y) requirements
+
+The Rivet table is built to follow the WAI-ARIA authoring standards. It is marked up with the appropriate ARIA attributes required to meet the [ARIA Authoring Practices standards](http://w3c.github.io/aria-practices/).
+
+### Labeling
```

Comment:
Let's include a note here that the following docs are for building a custom "table" using `div`s and `span`s, and not for using the `table` element or the examples in this doc, and include a link to the original doc: https://w3c.github.io/aria-practices/#table